gamemodes/terrortown/gamemode/client/cl_voice.lua

```diff
@@ -25,6 +25,48 @@ local MutedState
 
 g_VoicePanelList = nil
 
+local function VoiceTryEnable()
+	if not hook.Run("TTT2CanUseVoiceChat", LocalPlayer()) then return end
+
+	if not VOICE.IsSpeaking() and VOICE.CanSpeak() then
+		VOICE.IsTeam = false
```

Comment:
Vars should begin with small letter (isTeam). If this can be changed, it would be consistent

Reply:
there are loads of variables in Fabians style in our code though. We should define a coding style guideline

Reply:
Coding style already got defined. Maybe we need to sit together and talk about it. Making a paper could help too, but instead, I would use this time to improve my glua auto formatter / beautifier haha ^^

Reply:
There's a lot of instances of CamelCase instead of camelCase in the source code. It's just very inconsistent over all. Now that I know that you prefer camelCase, Ill go for it